notebooks/evaluation_2023.07/Ensemble Evaluation Model 1.ipynb

```diff
@@ -755,10 +755,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "url = f\"http://data-service.staging.terarium.ai/models/{res.json()['id']}\"\n",
+    "# url = f\"http://data-service.staging.terarium.ai/models/{res.json()['id']}\"\n",
     "\n",
-    "res = requests.get(url)\n",
-    "res.json()"
```

Comment:
In this notebook a number of things are commented out. Shouldn't we preserve those?